src/themes/dspace/styles/_global-styles.scss

```diff
@@ -17,7 +17,7 @@
     background-color: var(--bs-primary);
   }
 
-  h4 {
+    label {
```

Comment:
Why have you removed the default style for `<h4>`?  This change likely should be undone unless there's a clear reason why this was done.